mappings/net/minecraft/client/model/ModelPart.mapping

```diff
@@ -8,7 +8,11 @@ CLASS net/minecraft/class_630 net/minecraft/client/model/ModelPart
 	FIELD field_3665 visible Z
 	FIELD field_3674 roll F
 	FIELD field_3675 yaw F
+	METHOD <init> (Ljava/util/List;Ljava/util/Map;)V
+		ARG 1 cuboids
+		ARG 2 children
 	METHOD method_17138 copyPositionAndRotation (Lnet/minecraft/class_630;)V
```

Comment:
```suggestion
	METHOD method_17138 copyTransform (Lnet/minecraft/class_630;)V
```

Reply:
That was existing but yours is probably better